core/src/main/java/com/linecorp/armeria/common/stream/DefaultStreamMessageDuplicator.java

```diff
@@ -938,7 +937,7 @@ private int removeElements() {
                 if (!(o instanceof CloseEvent)) {
                     removedLength += signalLengthGetter.length(o);
                 }
-                PooledObjects.close(o);
+                StreamMessageUtil.closeOrAbort(o);
```

Comment:
When would `PooledObjects.close()` not enough?

Reply:
Replied at https://github.com/line/armeria/pull/3524/files#r627223166 😀